rental.rb

```diff
@@ -0,0 +1,14 @@
+require_relative './person'
+require_relative './book'
+class Rental
+  attr_reader :person, :book
```

Comment:
- [OPTIONAL] `Getters and Setters` for `@book and @person` instance variables should be present, like `attr_accessor :date, :book, :person`. Please kindly fix that.

Reply:
Thank you for approving even though the project lacks some data. I will fix that right now! Thank you very much